Minio/DataModel/ServerSideEncryption.cs

```diff
@@ -67,6 +69,11 @@ public SSEC(byte[] key)
             }
             this.key = key;
         }
+
+        public string GetCustomerKeyMD5()
+        {
+            return utils.getMD5SumStr(this.key);
+        }
```

Comment:
Didn't we decide in previous reviews that this method is unnecessary

Reply:
I've not pushed the resolved conflicts files yet. I will do it soon.

Reply:
not fixed